v3/cypress/support/commands.ts

```diff
@@ -115,6 +116,9 @@ Cypress.Commands.add("dragAttributeToTarget", (source, attribute, target, target
     case ("headerDivider"):
       target_el = el.headerDivider
       break
+    case('webView'):
```

Comment:
```suggestion
    case ("webView"):
```
For consistency and to eliminate the lint warning. If you were feeling motivated, for extra credit you could remove the gratuitous parentheses throughout the switch statements in this function, but that's not required.